Protected Node 7.x-1.0

dzsabo opened the pull request.

dzsabo committed:
Protected Node 7.x-1.0

czigor commented:
Mire lenne ez a modul használva? Az ügykezelőben lehet privát projektet létrehozni, amiben megadhatod, hogy ki férhet hozzá a tartalomhoz.

czigor commented:
OK, jó lesz, csak annyi, hogy a contrib modulok a sites/all/modules/contrib mappában vannak.

Úgy látom, a modul eléggé belemászik a fájlkezelésbe (ld. https://www.drupal.org/node/2661316), így alaposan le kéne tesztelni a jegyvásárlást és általában a fájlfeltöltést, hogy működik-e vele.

dzsabo committed:
Protected nodes to modules/contrib/ and testing

With Protected node module on, I tested on the dev site
- ticket purchasing system
- file upload and access (ones with and without a node associated to
them, both puclic and hidden drupal files, both with authenticated and
anonymous user)

dzsabo commented:
Egy jódarabig próbálgattam a fájlokkal a dolgokat, de semmi hibát nem találtam. A jegyvásárlás is működik.
